studio/src/entities/filters.js

```diff
@@ -0,0 +1,9 @@
+export default class MasFilters {
```

Comment:
Right now there is no purpose, because filters are not implemented. But when they are, this can be a place where everything about them can be handled, like `MasSearch` below.